WhatBackend.EmailRenderService/WhatBackend.EmailRenderService/Services/Interfaces/IMessageTemplateService.cs

```diff
@@ -5,8 +5,6 @@ namespace WhatBackend.EmailRenderService.Services.Interfaces
 {
     public interface IMessageTemplateService
     {
-        string GetAccountApprovedTemplate(AccountApprovedEvent message);
-
-        string GetRegistrationSuccessTemplate(RegistrationSuccessEvent message);
+        string GetEmailNotifyTemplate(string message);
```

Comment:
Why do you delete the other templates? Will we use one temlate for all notifications?

Reply:
> Why do you delete the other templates? Will we use one temlate for all notifications?

There is too many HTML code duplicates.
It's more efficient to use one template for all messages.
You'll need to set your message for a template at handler's method.

Reply:
Understood.